src/components/navigation-header/components/product-page-content/product-page-content.module.css

```diff
@@ -22,9 +22,4 @@
 .productLogoLink {
   composes: g-focus-ring-from-box-shadow-dark from global;
   border-radius: 5px;
-
-  /* On mobile/tablet, this element comes first */
-  &:not(:first-child) {
```

Comment:
This wasn't actually the first child on mobile so it wasn't working as expected.